pkg/envoy/registry/registry.go

```diff
@@ -15,22 +13,40 @@ func NewProxyRegistry(mapper ProxyServiceMapper, msgBroker *messaging.Broker) *P
 	}
 }
 
-// RegisterProxy implements MeshCatalog and registers a newly connected proxy.
+// RegisterProxy registers a newly connected proxy.
 func (pr *ProxyRegistry) RegisterProxy(proxy *envoy.Proxy) {
-	pr.connectedProxies.Store(proxy.UUID.String(), connectedProxy{
-		proxy:       proxy,
-		connectedAt: time.Now(),
-	})
+	pr.connectedProxies.Store(proxy.UUID.String(), proxy)
 	log.Debug().Str("proxy", proxy.String()).Msg("Registered new proxy")
 }
 
+// GetConnectedProxy loads a connected proxy from the registry.
+func (pr *ProxyRegistry) GetConnectedProxy(uuid string) (*envoy.Proxy, bool) {
```

Comment:
Can we drop the bool return in favor of the caller checking for nil proxy?

Reply:
done!